tests/Doctrine/Tests/Models/Cache/Action.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Doctrine\Tests\Models\Cache;
+
+use Doctrine\Common\Collections\ArrayCollection;
+
+/**
+ * @Entity
+ * @Table("cache_action")
+ */
+class Action
+{
+    const CLASSNAME = __CLASS__;
+
+    /**
+     * @Id
+     * @GeneratedValue
+     * @Column(type="integer")
+     */
+    protected $id;
+
+    /**
+     * @Column
+     */
+    private $name;
+
+    /**
+     * @OneToMany(targetEntity="Token", cascade={"persist", "remove"}, mappedBy="action")
+     */
+    private $tokens;
+
+    public function __construct($name)
+    {
+        $this->name = $name;
+        $this->tokens = new ArrayCollection();
+    }
+
+    public function addToken(Token $token)
```

Comment:
The `addToken` method is used to create easily the link between "action" and "token" instances